### ISSUE TITLE
Questions about validation accuracy

### ISSUE THREAD

**Issue by CHANTILLY2023** [whyandbecause/SDRNet#1]
Thank you for your outstanding work. I downloaded the pre-training weights and pvt_v2_b2 backbone network weights as instructed on the homepage, and never reported any error during the run. However, when the test size is set to 384, its eval result is a bit different from the paper as follows, which confuses me and I would like to ask why? My environment is RTX3090, pytorch1.9.1.

##################################################
Model:SDRNet,Dataset:CAMO_S||Smeasure:0.847; meanEm:0.898; wFmeasure:0.768; MAE:0.062; fnr:0.175;adpEm:0.904; meanEm:0.898; maxEm:0.919; adpFm:0.805; meanFm:0.81; maxFm:0.835
##################################################
Model:SDRNet,Dataset:CHAMELEON_S||Smeasure:0.887; meanEm:0.929; wFmeasure:0.802; MAE:0.034; fnr:0.132;adpEm:0.934; meanEm:0.929; maxEm:0.946; adpFm:0.824; meanFm:0.835; maxFm:0.869
##################################################
Model:SDRNet,Dataset:COD10K_S||Smeasure:0.843; meanEm:0.907; wFmeasure:0.721; MAE:0.03; fnr:0.187;adpEm:0.897; meanEm:0.907; maxEm:0.924; adpFm:0.73; meanFm:0.757; maxFm:0.793
##################################################
Model:SDRNet,Dataset:NC4K_S||Smeasure:0.869; meanEm:0.912; wFmeasure:0.79; MAE:0.042; fnr:0.159;adpEm:0.918; meanEm:0.912; maxEm:0.929; adpFm:0.819; meanFm:0.827; maxFm:0.851
##################################################

**Comment by whyandbecause** [comment]
Thanks for your interest in our work! It seems like a change in the code might have happened during ablation experiments or code organization. After a thorough check, I found the issue to **line 61 in the SDRNet.py**, the correct version should be **y4 = self.cv9(self.cv8(torch.cat(glfs, 1)))**. We have uploaded the updated file, thank you  for your feedback on this issue!

**Comment by CHANTILLY2023** [comment]
How many hours does the training take again please?

**Comment by CHANTILLY2023** [comment]
> Thanks for your interest in our work! It seems like a change in the code might have happened during ablation experiments or code organization. After a thorough check, I found the issue to **line 61 in the SDRNet.py**, the correct version should be **y4 = self.cv9(self.cv8(torch.cat(glfs, 1)))**. We have uploaded the updated file, thank you for your feedback on this issue!

Thank you for the explanation, and I would also like to ask how long is the training?

**Comment by whyandbecause** [comment]
> > Thanks for your interest in our work! It seems like a change in the code might have happened during ablation experiments or code organization. After a thorough check, I found the issue to **line 61 in the SDRNet.py**, the correct version should be **y4 = self.cv9(self.cv8(torch.cat(glfs, 1)))**. We have uploaded the updated file, thank you for your feedback on this issue!
> 
> Thank you for the explanation, and I would also like to ask how long is the training?

It takes about 12 hours！

**Comment by CHANTILLY2023** [comment]
THANK YOU VERY MUCH!!!

> > > Thanks for your interest in our work! It seems like a change in the code might have happened during ablation experiments or code organization. After a thorough check, I found the issue to **line 61 in the SDRNet.py**, the correct version should be **y4 = self.cv9(self.cv8(torch.cat(glfs, 1)))**. We have uploaded the updated file, thank you for your feedback on this issue!
> > 
> > 
> > Thank you for the explanation, and I would also like to ask how long is the training?
> 
> It takes about 12 hours！

THANK YOU VERY MUCH!!!
